src/Worker/DBClean.php

```diff
@@ -21,18 +21,25 @@ public static function execute($stage = 0) {
 		$days = Config::get('system', 'dbclean-expire-days', 0);
 
 		if ($stage == 0) {
-			for ($i = 1; $i <= 10; $i++) {
-				// Execute the background script for a step when it isn't finished.
-				// Execute step 8 and 9 only when $days is defined.
-				if (!Config::get('system', 'finished-dbclean-'.$i, false) && (($i < 8) || ($i > 9) || ($days > 0))) {
-					Worker::add(PRIORITY_LOW, 'DBClean', $i);
-				}
-			}
+			self::forkCleanProcess();
 		} else {
 			self::removeOrphans($stage);
 		}
 	}
 
+	/**
+	 * @brief Fork the different DBClean processes
+	 */
+	private static function forkCleanProcess() {
+		for ($i = 1; $i <= 10; $i++) {
+			// Execute the background script for a step when it isn't finished.
+			// Execute step 8 and 9 only when $days is defined.
+			if (!Config::get('system', 'finished-dbclean-'.$i, false) && (($i < 8) || ($i > 9) || ($days > 0))) {
```

Comment:
`$days` is undefined here. You need to bring over the following line:
```php
$days = Config::get('system', 'dbclean-expire-days', 0);
```

Reply:
Done